clients/databricks_client.py

```diff
@@ -0,0 +1,79 @@
+import requests
+import time
+import os
+
+class DatabricksClient:
+  def __init__(self, logger):
+    self._logger = logger
+    self._client_id = os.getenv('DATABRICKS_CLIENT_ID')
+    self._client_secret = os.getenv('DATABRICKS_CLIENT_SECRET')
+    self._host = "https://" + os.getenv('DATABRICKS_HOST')
+    self._token_url = self._host + "/oidc/v1/token"
+
+  def apply_review_results(self, table_name, accepted_classifications, rejected_classifications, token=None, token_expiration=None):
```

Comment:
does it make sense to pass token expiration around everywhere, or should we just set it as an instance property of this `DatabricksClient` whenever we refresh the token?